ibm/mas_devops/roles/mariadb/vars/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
if we don't have any vars we don't need this file, there's no need for an empty one